components/skeleton/List.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import omit from 'omit.js';
+import classNames from 'classnames';
+import Element, { SkeletonElementProps } from './Element';
+import { ConfigConsumer, ConfigConsumerProps } from '../config-provider';
+
+export interface SkeletonListProps extends Omit<SkeletonElementProps, 'size'> {
+  size?: 'large' | 'small' | 'default';
+}
+
+const SkeletonList = (props: SkeletonListProps) => {
+  const renderSkeletonList = ({ getPrefixCls }: ConfigConsumerProps) => {
+    const { prefixCls: customizePrefixCls, className, active } = props;
+    const prefixCls = getPrefixCls('skeleton', customizePrefixCls);
+    const otherProps = omit(props, ['prefixCls']);
+    const cls = classNames(prefixCls, className, `${prefixCls}-element`, {
+      [`${prefixCls}-active`]: active,
+    });
+    return (
+      <div className={cls}>
+        <Element prefixCls={`${prefixCls}-list`} {...otherProps} />
```

Comment:
I think if you want to use Element you should add this css class into css file.

For now, your implement is a empty span. 
<img width="518" alt="image" src="https://user-images.githubusercontent.com/29879298/84042787-1941e380-a9d8-11ea-82cb-a3d1dfb4e568.png">

You can preview it by `npm start`.

Reply:
Added css

Reply:
I think the SkeletonList  should look like List component. It seems your impl only one element.

You can refers List component API and shape.